WebViewControl/WebView.cs

```diff
@@ -492,6 +501,11 @@ public partial class WebView : IDisposable {
                     JsExecutors.Remove(indexedExecutorKey);
                 }
             }
+        }
+
+        private void UnsubscribeChromiumJavascriptContextEvents() {

```

Comment:
consider moving this inline... no need to have a separate method (not reused, no counterpart initialization)